tests/crypto_entropy/main.c

```diff
@@ -0,0 +1,43 @@
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "warnp.h"
+
+#include "crypto_entropy.h"
+
+/* This must match the value in crypto/crypto_entropy.c! */
+#define RESEED_INTERVAL 256
+
+int
+main(int argc, char * argv[])
+{
+	uint8_t buf[32];
+	size_t i;
+
+	WARNP_INIT;
+
+	(void)argc;
+	(void)argv;
+
+	/* Get entropy enough times to trigger a reseed. */
+	for (i = 0; i < RESEED_INTERVAL + 1; i++) {
+		if (crypto_entropy_read(buf, 32))
+			goto err0;
+	}
+
+	/* Print final random value. */
+	for (i = 0; i < 32; i++) {
```

Comment:
Shouldn't we be printing a pass/fail result?  Not that we have any way to recognize if random numbers are random, but at least we know if there was an internal error.